test/e2e_node/gke_environment_test.go

```diff
@@ -303,6 +303,7 @@ func checkDockerSeccomp() error {
 
 // checkDockerStorageDriver checks whether the current storage driver used by
 // Docker is overlay.
+//lint:ignore U1000 referenced in other functions in this file.
```

Comment:
if it is, why isn't the test passing?

Reply:
It's being called here, https://github.com/kubernetes/kubernetes/blob/4a25f6ffcdbf97859c11f38cc2f78d11661a1e67/test/e2e_node/gke_environment_test.go#L365
However, somehow staticcheck complain that it's unused. Maybe its related with several issues around `U1000` checks, e.g. [this](https://github.com/dominikh/go-tools/issues/726) and [this](https://github.com/dominikh/go-tools/issues/768) which might be solved by this [commit](https://github.com/dominikh/go-tools/commit/5cfc85b70e7b778eb76fd7338e538d7c9af21e4e) which I think is part of release 2020.1.5.

Reply:
So the root cause if the skipped testcase. And the question is whether we need to keep it skipped or remove. Test is created back in 2017 https://github.com/kubernetes/kubernetes/pull/49443. 

Let me check whether it's an overlay driver now...

If not, perhaps better fix would be to not skip the test, but instead check that this is NOT an overlay driver.

Reply:
ok. it seems that the better fix will be to remove this line: 

`e2eskipper.Skipf("GKE does not currently require overlay")`

GKE seems to support overlay2 now.

Reply:
ok, will do that...